src/addon/cc-addon-backups.js

```diff
@@ -333,12 +332,12 @@ export class CcAddonBackups extends LitElement {
 
   static get styles () {
     return [
-      defaultThemeStyles,
       skeletonStyles,
       linkStyles,
       // language=CSS
       css`
         :host {
+          color: var(--cc-color-text-default);
```

Comment:
I never set default value for this var because if it not set, it would just inherit its parent color, which I think is good enough ?